.gitlab-ci.yml

```diff
@@ -77,6 +78,7 @@ test_subscriptions:
 setup_fusillade:
   stage: fusillade
   script:
+    - date && date -u
```

Comment:
Is there a reason to call `date` again here?

Reply:
I would like to have both local timezone, and UTC because our logging system records events in UTC.

Reply:
The command `date && date -u` already called in before_script on line 21. Why call it again here?

Reply:
Ah! i see what you mean; removing.